### ISSUE TITLE
fmgr_sys_hitcount does not return task id to be able to fetch the result of the hitcount command

### ISSUE THREAD

**Issue by xav-git** [fortinet-ansible-dev/ansible-galaxy-fortimanager-collection#56]
With following ansible code:

  tasks:
   - name: Count_HITCount
     fmgr_sys_hitcount:
        bypass_validation: False
        enable_log: 'true'
        sys_hitcount:
           adom: DATA
           pkg: FW001
        register: r

   - name: Poll the task
     fmgr_fact:
       facts:
        selector: 'task_task'
       params:
        task: '{{ r.meta.response_data.task }}'
     register: taskinfo
     until: taskinfo.meta.response_data.percent == 100
     retries: 30
     delay: 5

Error Message:  "The task includes an option with an undefined variable. The error was: 'list object' has no attribute 'task'
When displaying r we have the same problem. 

Expected Behavior:
The hitcount command should provide a task id as result to be able to retrieve the result of the hitcount command.

**Comment by jpforcioli** [comment]
I can reproduce the issue.
`taskid` is returned by FortiManager, but not in the usual `data` dict:

```json
{
  "id": 6,
  "result": [
    {
      "status": {
        "code": 0,
        "message": "OK"
      },
      "taskid": 363,
      "url": "sys/hitcount"
    }
  ]
}
```
`fmgr_sys_hitcount` but also `fmgr_generic` failed at catching it.

**Comment by jpforcioli** [comment]
Internal Reference: FMG: `0870456`

**Comment by dux-fortinet** [comment]
Thanks for pointing it out. This problem is fixed in FortiManager Ansible Collection 2.2.0.

You can use `r.meta.response_data.task` to get the task id now.
